packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -843,8 +875,10 @@ class _ViewContentState extends State<_ViewContent> {
       return MediaQuery.removePadding(
         context: context,
         removeTop: true,
-        child: ListView(
-          children: suggestions.toList()
+        child: ListView.builder(
+          shrinkWrap: true,
+          itemCount: suggestions.length,
+          itemBuilder: (BuildContext context, int index) => suggestions.elementAt(index),
```

Comment:
Why we have this change here?

Reply:
Do you mean making the ListView lazy? Is that not best practice?

Reply:
Although it looks like making the list lazy here doesn't actually help performance. In testing with a lot of suggestion items there are still performance issues so maybe we can revert this.

Reply:
(Setting shrinkWrap to false doesn't help either)